### ISSUE TITLE
Edit (some) fields of flow after upload?

### ISSUE THREAD

**Issue by PGijsbers** [openml/openml-python#896]
I noticed I have some uploaded flows for which I would like to add a `custom_name`. I'm currently unable to do so through the Python API (or at all). This gives rise to the following questions:

 - Should we support filling in the `custom_name` field later if left empty initially?
 - Are there other attributes for which this makes sense?
 - Should we support editing `custom_name` (or other fields) after uploading?
 - Should we look at other entities as well to see what we might want to be editable?

Of course it is important to preserve consistency. Especially if other users start using your flow, you don't want it to suddenly represent something else. 

Personally I feel that providing a `custom_name` after not having one at upload time is fine. I would say editing `custom_name` is OK too, considering we auto-generate them with the scikit-learn extension for some flows (and if that goes wrong, editing is probably preferred?).

Other fields that seem candidates for editing to me are: `tags`, `language` and `binary_url` (probably not md5/format).

I'm not sure to which extend this is a design issue or just openml-python.

Paging @mfeurer @janvanrijn @joaquinvanschoren

**Comment by joaquinvanschoren** [comment]
This requires a new REST API endpoint.

We then need to discuss for which fields we want to allow editing.
For inspiration, see the data edit pull request by Sahi:
https://github.com/openml/OpenML/pull/1044

**Comment by PGijsbers** [comment]
PR related to that data edit change for openml-python: #935

**Comment by mfeurer** [comment]
Hey, is this issue resolved? The two PRs above are both merged

**Comment by PGijsbers** [comment]
No. The changes were only made to allow edits to dataset entities. OpenML Python still does not support changing the flows as far as I can tell.

**Comment by Prachi194agrawal** [comment]
Hello everyone,

I've opened **Pull Request #1527** (linked above) to address this issue by implementing the client-side functionality for editing flow metadata.

The new function, **`openml.flows.edit_flow`**, allows authenticated users to update the following non-critical fields on an existing flow:

* **`custom_name`**
* **`tags`**
* **`language`**
* **`description`**

This implementation relies on the existence of the corresponding **`flow/edit` REST API endpoint**.

I believe this completes the work needed on the `openml-python` side to close Issue #896. Could a maintainer please review PR #1527?

Thanks!
